tests/test_dataclasses.py

```diff
@@ -2944,3 +2944,29 @@ class Y:
         x: Union[X, None]
 
     assert Y(x={'y': None}).x.y is None
+
+@pytest.mark.parametrize("field_fn_unset", [Field, dataclasses.field])
+@pytest.mark.parametrize("field_fn2", [Field, dataclasses.field])
+def test_annotated_with_field_default_factory( field_fn_unset, field_fn2) -> None:
```

Comment:
Without entering into the details of the fix, perhaps you might want to call `make format` to let `ruff` fix formatting for you...

Reply:
Weirdly, when I run `make format` I only get the following output:
```
PDM, version 2.17.1
pdm run ruff check --fix pydantic tests docs/plugins
All checks passed!
pdm run ruff format pydantic tests docs/plugins
87 files left unchanged
```
It does not format the unformatted test file, so I have to explicitly format it: 
```
$ pdm run ruff format .\tests\test_dataclasses.pytaclasses.py  test_datetime.py
1 file reformatted                              
```
It does work for files under `pydantic/` though.

Reply:
Interesting 🤔 might be worth a check as `pdm run ruff format pydantic tests docs/plugins` does consider `tests`.
`